samples/winapi_filefind/main.c

```diff
@@ -10,10 +10,15 @@ int main(void)
 {
     XVideoSetMode(640, 480, 32, REFRESH_DEFAULT);
 
+    // Create a variable for WinAPI error checking
+    DWORD error;
+
     // Mount C:
     BOOL ret = nxMountDrive('C', "\\Device\\Harddisk0\\Partition2\\");
     if (!ret) {
-        debugPrint("Failed to mount C: drive!\n");
+        // There was an error. We can get more information about an error from WinAPI code using GetLastError()
+        error = GetLastError();
+        debugPrint("Failed to mount C: drive! Reason: %x\n", error);
         Sleep(5000);
         return 1;
```

Comment:
This return value is now different compared to the other handling handling. I suggest to remove these two lines and replace them with a goto to the `Sleep()`-loop (you can name the label `end` or something).